package.json

```diff
@@ -15,8 +15,15 @@
     "clean": "cd packages/ui5-app && yarn clean",
```

Comment:
little hint: once your PR is in, I will switch here to the yarn workspace syntax, e.g.:

`yarn workspace ui5-app clean`

Reply:
roger that